ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -836,6 +838,13 @@ export default class ConfirmTransactionBase extends Component {
         txData.metadata.note = noteText;
       }
 
+      if (
+        smartTransactionsOptInStatus &&
```

Comment:
If we're trying to detect if this is a smart transaction, should we be using the `getSmartTransactionsEnabled` selector as it also checks the feature flag so if the service is live?

Reply:
Thanks for the quick response @matthewwalsh0 ! I appreciate it; I didn't know that it existed. I'll do the changes